examples/kubernetes-in-docker/README.md

```diff
@@ -0,0 +1,333 @@
+# Deploying Applications that Use Conjur Kubernetes Authentication on Kubernetes-in-Docker (KinD)
+
+This directory provides a step-by-step guide for deploying a simple
```

Comment:
So I ran into several issues with running these scripts, because I didn't have an existing Helm deployment. I misunderstood this demo as one that could create a deployment then perform some action upon it, i.e. end-to-end. This was definitely my user error, but I think we could add a note or make it clearer early on that you can't just run the `start` script, as with some demos.

Reply:
I tried to make it clear in the `Prerequisites` section of the README.md that this assumes that you have a Helm installed Conjur cluster.

I could make this fully end-to-end, including the Helm install of Conjur... but my original intent was to have a bunch of example demos, one for each Kubernetes platform (i.e. one for KinD, one for GKE, one for OpenShift), with each demo assuming that you've already Helm installed Conjur.

Let me try to make this fully E2E, see what it looks like.